app/controllers/editions/two_eye_outcomes_controller.rb

```diff
@@ -0,0 +1,46 @@
+class Editions::TwoEyeOutcomesController < BaseController
+  def new
+    @edition = Edition.find(params[:id])
+    render :new
+  end
+
+  def create
+    @edition = Edition.find(params[:id])
+    if two_eye_outcome_supplied?
```

Comment:
I was thinking we could change this to `second_pair_of_eyes_outcome_supplied?`, but actually, I think the prefix is redundant here anyway, as the only outcomes we're talking about are 2i outcomes, so can just be `outcome_supplied?` etc.